src/Components/Charts/BarChart/BarChart.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { BarChart } from '../Charts';
+
+
+function CustomBarChart(props) {
+    const { data, title, childKey } = props;
+    const [barChartData] = useState(JSON.parse(JSON.stringify(data)));
```

Comment:
If you wish another method, could use const [barChartData] = useState({ ...data });

Reply:
If I just replace will not work because const [barChartData] = useState(JSON.parse(JSON.stringify(data))); return an object while const [barChartData] = useState({ ...data }); return an array and the chart expects an object